### ISSUE TITLE
Symlink python->python3 im Dockerfile

### ISSUE THREAD

**Issue by mgmax** [fau-fablab/website#8]
Ich halte es für falsch, dass `python` auf irgendwas anderes als `python2` zeigt. Wenn du genug suchst, wirst du sicher einen Standard finden der das auch explizit sagt.

**Comment by patkan** [comment]
So weit ich das beurteilen kann, gibt es zwar keinen Standard, aber unter Umständen macht man die Betriebssystemtools kaputt. Je nachdem ob die auf Python 3 portiert sind. Da stellt sich dann die Frage, wie das bei fedora ist.

**Comment by mgmax** [comment]
https://www.python.org/dev/peps/pep-0394/
`for the time being, all distributions should ensure that python refers 
to the same target as python2 .`

Skripte sollten korrekt python3 oder python2 als Interpreter angegeben 
haben.

**Comment by sedrubal** [comment]
Ja, ich glaube, man braucht das auch nicht unbedingt, aber es hat mir beim Frickeln geholfen, da Pakete nur für Python3 installiert waren und wenn man `python` verwendet hat, gabs immer nur Fehlermeldungen...

**Comment by sedrubal** [comment]
naja das Problem ist, dass Django kein Skript ist, sondern ein komplexer Haufen...

**Comment by mgmax** [comment]
Ansonsten hast du undefiniertes Verhalten, wenn eigentlich für python2 
gedachte Skripte mit python3 ausgeführt werden. Das ist noch schlechter.

**Comment by sedrubal** [comment]
Die Idee war, alles python3 only zu machen, weil es sonst echt kompliziert wird (mittlerweile bin ich mir aber nicht mehr sicher, ob man wirklich python3 haben will)...

**Comment by sedrubal** [comment]
Jo, also ich denke, die Lösung wäre, alles auf Python2 umzubauen?

**Comment by mgmax** [comment]
Ohne es django-spezifisch genau beurteilen zu können: Tendentiell ist python3 schon sinnvoller, wenn ich z.B. an die Unicode-Qualen bei python2 denke. Der Symlink ist trotzdem in fast allen Fällen Unsinn, stattdessen sollte man die betreffenden Skripte auf `#!/usr/bin/env python3` ändern, damit sie in normaler Linuxumgebung auch funktionieren.

**Comment by sedrubal** [comment]
Achso stimmt, unicode... Allerdings hatte ich bis jetzt noch keinen wirklichen "Kontakt" mit Python, das ist alles recht Highlevel und mehr Provisioning Frickelage als Python Programmieren (vorausgesetzt man verwendet bestehende Plugins).
Bis auf `manage.py` gibt es eig. keine Skripte.
Python3 und 2 mischen fände ich nicht so schön, wie hart hin frickeln ;)
